matrix/build.gradle.kts

```diff
@@ -26,7 +26,11 @@ val dockerWorkingDir = project.buildDir.resolve("docker")
 
 val buildTestImagesTask = tasks.create("buildTestImages") {
```

Comment:
As these are now built by Otel, can we remove this?

Reply:
And leave only proprietary?

Reply:
wait, liberty is also not present in otel?

Reply:
Removed template and matrix definition, updated README.